frontend/test/lit/test_device_api.py

```diff
@@ -65,8 +65,9 @@ def preprocess(self, execution_config: Optional[ExecutionConfig] = None):
 def test_circuit():
     """Test a circuit compilation to MLIR when using the new device API."""
 
-    # CHECK:    quantum.device["[[PATH:.*]]librtd_lightning.{{so|dylib}}", "dummy.remote", "{'shots': 2048}"]
-    dev = DummyDevice(wires=2, shots=2048)
+    # CHECK
+    # quantum.device["[[PATH:.*]]librtd_null_device.{{so|dylib}}", "NullDevice", "{'shots': 2048}"]
```

Comment:
Please revert these changes. These CHECK: comments are the actual test.

```suggestion
    # CHECK:    quantum.device["[[PATH:.*]]librtd_lightning.{{so|dylib}}", "dummy.remote", "{'shots': 2048}"]
```

Reply:
Oops. Sorry. But the codefactor is complaining about the line size.